crates/proto/src/op/edns.rs

```diff
@@ -113,6 +113,18 @@ impl Edns {
     pub fn set_option(&mut self, option: EdnsOption) {
         self.options.insert(option);
     }
+
+    pub(crate) fn len(&self) -> u16 {
+        1 //   Name::root
```

Comment:
I wonder if we should consider adding some consts for these things, or perhaps instead use `std::mem::size_of` It would make the code a bit more self-documenting. Thoughts?

I know we have a similar practice elsewhere in the code, maybe I'll file a cleanup issue for this.

Reply:
Hmm, could we use `EncodedSize` for this?
Let's do this in another PR anyway